packages/format/tests/index.test.ts

```diff
@@ -7,6 +7,16 @@ it('first argument', () => {
   expect(format('')).toBe('')
   expect(format([])).toBe('[]')
   expect(format({})).toBe('{}')
+  expect(format(Object.create(null))).toBe('{}')
```

Comment:
In Node.js, this is printed as `[Object: null prototype] {}` 🤔

Reply:
Thank you very much, I didn't notice this, I just tested the case on the browser, but I just tried it in Deno and it looks like this:

![image](https://user-images.githubusercontent.com/41503212/195810653-cda2dcda-2e45-4f20-9bf6-f2f4e0d7a9b0.png)

Ummm, I don't know which one is better for us.